keps/sig-storage/1698-generic-ephemeral-volumes/README.md

```diff
@@ -51,10 +51,10 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 - [X] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
 - [X] (R) Graduation criteria is in place
 - [X] (R) Production readiness review completed
-- [ ] Production readiness review approved
-- [ ] "Implementation History" section is up-to-date for milestone
-- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
-- [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+- [X] Production readiness review approved
```

Comment:
@wojtek-t : I am assuming that the review we did for beta still applies because nothing changed. Correct?

Reply:
Some answers to the PRR mention things like "pending beta" or "TBD for GA". Can you update those answers?

Reply:
Good point. I went through the document and I hopefully found all those places.

Regarding SLO and "needs more discussion": I think what was proposed ("same rate as for stand-alone PVCs") is reasonable and uncontentious, so I removed the "needs more discussion".

We discussed on Slack that the existing pod startup test might be usable for generic ephemeral volumes. I checked that and only had to make some minor modifications to make it work - see https://github.com/kubernetes/perf-tests/pull/1879